### ISSUE TITLE
Regression with image

### ISSUE THREAD

**Issue by Neoplayer** [dotnet/machinelearning#5105]
How to concatenate image with some Single values? I was trying to convert image to `ExtractPixels` and `LoadRawImageBytes` but I can't find any way to concatenate this with single values.

**Comment by justinormont** [comment]
@Neoplayer: Can you also describe your ML task a bit more?

Sounds like you have an image column plus some numeric values. Then you're looking to predict a float value (regression).

This should work as @luisquintanilla describes. You should be use one of the pre-trained image models as a feature extractor for the images, and concatenate with your numeric features. Then feed that final features column to your regression trainer.

Example of the `DNNImageFeaturizer`: (will have to add the concatenate/regression trainer)
https://github.com/dotnet/machinelearning/blob/25f8c5bea58dd852b27aa1dec1e931e6e5509d01/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs#L44-L54

Alternatively you can use a custom TensorFlow model ([example](https://github.com/dotnet/machinelearning-samples/tree/ce4b22acd80de17385daa4b5722091d7f24d4c79/samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow)), or a custom ONNX model ([example](https://github.com/dotnet/machinelearning/blob/01537545f8495b98e291e45eb24e187656406dae/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs#L25-L31)) for your image featurized.

Another route is to use the `ImageClassificationTrainer`. This will refit a pre-trained model to your data. You'll have to bin your regression labels and feed it as multi-class. Then use the output scores from `ImageClassificationTrainer` as input to your concatenation to your final regression model ([model stacking)](https://en.wikipedia.org/wiki/Ensemble_learning#Stacking). `ImageClassificationTrainer` example: https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/DeepLearning_ImageClassification_Training
